### PR TITLE
issue560/gpl 712 as a user when the destination plate is ready i would like to scan the barcode into the limber pipeline and decide whether it will be processed as a 384 or 96 well plate

### LINKED ISSUE #560: GPL-712 As a user when the destination plate is ready I would like to scan the barcode into the limber pipeline and decide whether it will be processed as a 384 or 96 well plate
**User story**
When the destination plate is ready scan the barcode into the limber pipeline and at this stage the user will make the decision on whether it will be quad stamped or continue to be processed as a 96 well plate and then a library submission will be created automatically.

**Who are the primary contacts for this story**
e.g. John S (don't include surnames in public repos)

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] LTHR Cherrypick plate recognized by limber
- [ ] New plate presenter provides submission decision

***384-well***
- [ ] Add LTHR Cherrypick >  LTHR-384 RT relationship to existing pipeline
- [ ] LTHR-384 RT -> Quad stamp creator (Use existing?)

***96-Well***
Either:
- [ ] New PlateConversion creator
- [ ] LTHR RT -> Plate conversion creator
- [ ] Add LTHR Cherrypick >  LTHR RT relationship to existing pipeline

Or:
- [ ] Add LTHR Cherrypick >  LTHR PCR 1 relationship to existing pipeline
- [ ] Add LTHR Cherrypick >  LTHR PCR 2 relationship to existing pipeline
- [ ] Add LTHR Cherrpick versions of  LTHR RT robots

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

### PR DESCRIPTION
**Pull request by JamesGlover**
Add new front-end for LTHR pipeline

closes #560

- Add missing library types to limber pipelines
- Hotfix: PCR 2 pathways for new LT
- Use yaml anchors for shared filters
- Update Rubocop
- Extract statechangeless behaviour
- Fix comment
- Add outline of Submission Presenter
- Stock plate factory passed by default
- SequencescapeSubmission supports multiple asset_groups
- Submission presenter can generate submissions
- Update sequencescape-client-api
- Add submission presenter and creation
- Presenter documentation
- Robots can have multiple valid purposes
- Remove unused robot class
- Configure new links in LTHR pipeline
- Add QuadStamp robot
- Reformat line for readability
- LTHR RT needs to be LHTR RT-Q
- Add Primer panel templates
- Add uncreatable creator
- Remove old pipelines and library types
- Update documentation

### CODE REVIEW

File: app/controllers/concerns/sequencescape_submission_behaviour.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Module SequencescapeSubmissionBehaviour provides the ability to
+# generate submissions to controllers
+#
+# @author Genome Research Ltd.
+#
+module SequencescapeSubmissionBehaviour
```
Comment (stevieing) [inline app/controllers/concerns/sequencescape_submission_behaviour.rb:8]: This is good. Keeping it separate makes it transparent.

File: app/models/concerns/presenters/statemachine/submission.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_dependency 'presenters/statemachine'
+module Presenters::Statemachine
```
Comment (stevieing) [inline app/models/concerns/presenters/statemachine/submission.rb:4]: Nice. Again quite transparent. What is the rubocop problem. Can it be fixed?

File: app/models/labware_creators/uncreatable.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module LabwareCreators
```
Comment (stevieing) [inline app/models/labware_creators/uncreatable.rb:3]: Is an uncreatable creator an oxymoron!

**Review by stevieing — APPROVED** [review]
All good. Liberal use of configuration makes it much easier to understand.

File: app/models/concerns/presenters/statemachine/submission.rb (hunk 2 above)
Comment (JamesGlover) [reply app/models/concerns/presenters/statemachine/submission.rb:4]: In this case yes, I should just write some documentation! I'll handle that.

**Pull request merged**